setup.cfg

```diff
@@ -19,6 +19,12 @@ install_requires =
     srsly>=1.0.1,<3.0.0
     catalogue>=0.2.0,<3.0.0
 
+[options.entry_points]
+spacy_readers =
+    ml_datasets.imdb_sentiment.v1 = ml_datasets.spacy_readers:imdb_reader
+    ml_datasets.cmu_movies.v1 = ml_datasets.spacy_readers:cmu_reader
+    ml_datasets.dbpedia.v1 = ml_datasets.spacy_readers:dbpedia_reader
+
```

Comment:
Question is whether we want to have these in `spacy.readers` or some other registry. The type is slightly different. An actual `Reader` from spacy is `Callable[["Language", str], Iterable["Example"]]`, but these functions are `Dict[str, Callable[["Language", str], Iterable["Example"]]]` as they read the `train` and `dev` bit in one go, as opposed to `spacy.Corpus` which reads only one at the time.

Reply:
Decided after internal conversation that we'll keep both types in "readers".